PublishParasoftTestResults/publishtestresults.ts

```diff
@@ -269,9 +279,44 @@ function checkStaticAnalysisViolations(sarifReports: string[], index: number) {
     }
 }
 
+function loadSettings(localSettingsPath : string) {
```

Comment:
Run "tsc" under "PublishParasoftTestResults" folder and resolve the issue related to this line.

Reply:
I see no issue when run "tsc" on my machine.

Reply:
I think the error I saw maybe related to my lower node version in my home pc. We can ignore this comment.